ui/fisher/fisher.js

```diff
@@ -147,8 +146,8 @@ class Fisher {
       },
     };
 
-    this.ui = new FisherUI(element, Options);
-    this.seaBase = new SeaBase(Options);
+    this.ui = new FisherUI(element, this.options);
```

Comment:
I think the `options` parameter should be the first one? Just like others do.

Reply:
Done, thanks for the suggestion!